Contributing.md

```diff
@@ -0,0 +1,50 @@
+# Stakeholders
+
+- **Project Manager**: responsible for aligning task creation with broader program direction through discussion with task manager(s). May comment on tasks.
+- **Task Manager**: responsible for day-to-day interfacing with github. Has authority to create, edit, and close tasks. Has authority to upload and remove files from the repository.
```

Comment:
I've been thinking a bit on this role. I think that anyone should be able to do simple issue handling, at least anyone who has proved to be contributing. But I still think this role is useful for making sure there is a structure among the issues. Making sure newly created issues have the correct data and are complete with no ambiguity etc.
Perhaps we should introduce a new role: Contributor. A contributor should be able to handle tasks as well, but there might be some limitations. E.g. cannot create new project. A contributor should also be able to send pull requests.